hw2/04-background.js

```diff
@@ -1 +1,32 @@
 // Add your code here
+const body = document.getElementById('background');
+const timeInterval = document.getElementById('interval');
+const changeColorBtn = document.getElementById('changeColorBtn');
+
+const colorChange = function randomColorChangeForBodyBackground() {
+  const r = Math.floor(Math.random() * 256);
```

Comment:
Since rgb numbers range from 0 to 255, I didn't get why to multiply by 256! Is there any specific reason to do so?

Reply:
Because 0 is an available option, Math.random() * 256 gives numbers 0-255.